.gitmodules

```diff
@@ -1,12 +1,9 @@
 [submodule "kucher/libraries/popcop"]
 	path = kucher/libraries/popcop
 	url = https://github.com/Zubax/popcop
-[submodule "kucher/libraries/construct"]
-	path = kucher/libraries/construct
-	url = https://github.com/construct/construct
 [submodule "kucher/libraries/dataclasses"]
 	path = kucher/libraries/dataclasses
 	url = https://github.com/ericvsmith/dataclasses
```

Comment:
This is no longer needed and should be deleted. Dataclasses are now part of the standard library.